### ISSUE TITLE
Selecting item with mouse and selection in middle setting causes weird highlighting when mouse button pressed for longer timespan

### ISSUE THREAD

**Issue by pcause** [martinrotter/rssguard#246]
using last dev build from yesterday.  i have checked to keep selection in middle.  i scroll see an item i want to click on and when I click i get several items highlighted, with the selected at the bottom

![image](https://user-images.githubusercontent.com/1071479/84907028-35ade200-b0a2-11ea-83f9-3e3b3f8de6b7.png)

**Comment by pcause** [comment]
more data.  select an item with mouse.  hit keystroke to open in external browser.  keystroke and not right click menu item.  then scroll down with mouse wheel, click another entry.

**Comment by martinrotter** [comment]
Are you pressing down some button on keyboard while clicking? I cannot reproduce the issue.

**Comment by pcause** [comment]
nope, not pressing down anything.  let me try again to reproduce, but the steps i mentioned in previous comment worked every time.  let me see if there is some step I missed.

**Comment by martinrotter** [comment]
I managed to reproduce, this bug happen if you click "slowly". You keep your mouse button pressed for a longer period of time, like half of second and so on.

**Comment by martinrotter** [comment]
Also, you can selected multiple messages just by mouse when pressed left mouse button and dragging, this is what causes this weird behavior.

**Comment by martinrotter** [comment]
OK, I pushed 3e8702fb.

Pls check if it works and let me know.

**Comment by pcause** [comment]
sees to work after a few hours of use
